scipy/interpolate/tests/test_bsplines.py

```diff
@@ -1094,59 +1151,6 @@ def make_interp_full_matr(x, y, t, k):
     return c
 
 
-### XXX: 'periodic' interp spline using full matrices
```

Comment:
Why removing this? Maybe better keep this and test your implementation against.
(if this version is buggy, fix it)